spec/controllers/openid_token_proxy/callback_controller_spec.rb

```diff
@@ -36,7 +36,11 @@
       context 'with no-op token acquirement hook' do
         it 'redirects to root' do
           OpenIDTokenProxy.configure_temporarily do |config|
-            config.token_acquirement_hook = proc { }
+            block = double('block')
+            config.token_acquirement_hook = proc { |token|
+              block.run(token)
+            }
+            expect(block).to receive(:run).with(instance_of(OpenIDTokenProxy::Token))
```

Comment:
Still think this should be using RSpec's standard block testing matchers, rather than the configurable block calling our own block and then verifying expectations on that throw-away block.

Any reason why the double doesn't suffice here?